FrEIA/modules/orthogonal.py

```diff
@@ -82,49 +139,32 @@ def __init__(self, dims_in, dims_c=[], n_reflections=1, fixed=False):
                 # init close to identity
                 init = torch.eye(self.width, self.n_reflections)
                 init += torch.randn_like(init) * 0.1
-            Vs = torch.unbind(init, dim=-1)
-            self.Vs = [nn.Parameter(V) for V in Vs]
-            for i, V in enumerate(self.Vs):
-                V.requires_grad = not self.fixed
-                self.register_parameter(f'V_{i}', V)
+            Vs = init.transpose(-1, -2)
```

Comment:
Is there a reason why the vectors are stored as tuples? ```_fast_h``` needs ```V``` to be a tensor.

Reply:
I don't think there was a particular reason.